src/style.css

```diff
@@ -0,0 +1,20 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+div {
```

Comment:
use class name for this, you may have more divs and other ones will have different appearance, this way you will need to override your styles